content/blog/kubecon-na-2021/index.md

```diff
@@ -0,0 +1,40 @@
+---
+path: '/kubecon-na-2021'
+title: 'Come meet us at Kubecon NA 2021'
+date: 2021-10-12T00:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'Announcements', 'Kubernetes']
+authors: ['Zain Asgar', 'Michelle Nguyen']
+emails: ['zasgar@pixielabs.ai', 'michelle@pixielabs.ai']
+featured: true
+---
+Pixie will be attending this year's Kubecon + Cloud NativeCon NA in Los Angeles! Pixie was accepted to CNCF’s Sandbox in June 2021 and we have just officially completed the [Sandbox onboarding process](https://github.com/cncf/toc/issues/674). As a newly fledged CNCF Sandbox project, we're excited to connect with the community.
+
+Come chat observability, eBPF, open-source, and more with Pixie’s core contributors at our booth in the [Project Pavilion](https://events.linuxfoundation.org/wp-content/uploads/2021/10/KubeCon_NA_2021_Collateral_17x11_v2.pdf) from 10:30am-4:30pm, on all event days.
+
+We will also be holding three events this year:
+- [**Pixie Happy Hour**](https://pixiehh.eventbrite.com/) (Tuesday 10/12, 8:30pm @ Spire 73)\
+Discuss Pixie, eBPF, and Kubernetes over appetizers, drinks, and sweeping views of Los Angeles from Spire 73, the tallest open-air bar in the Western hemisphere.
+
+- [**Project Office Hours**](https://kccncna2021.sched.com/event/nBuh/virtual-project-office-hours-pixie) (Wednesday 10/13, 3:30pm @ Online)\
+We're excited to meet everyone at our office hours! We welcome all questions about Pixie, whether they're questions about what Pixie is, or low-level questions about Pixie's internal workings.
+
+- [**Data Science for Infrastructure: Observe, Understand, Automate**](https://kccncna2021.sched.com/event/lV4c/data-science-for-infrastructure-observe-understand-automate-zain-asgar-natalie-serrino-new-relic) (Friday 10/15, 11:00am @ Concourse Hall 152)\
+Learn how Pixie's data can be transformed to high-level signals that can be used in a variety of use cases from Zain Asgar and Natalie Serrino (core contributors).
+
+Here are some of the events we'll be attending. We hope to see you at them, along with the many other exciting talks at Kubecon!
+
+**Wednesday, October 13**
+- 11:00 am: [Cloud Native and Observability Panel: The State of the Union](https://kccncna2021.sched.com/event/lV1N/cloud-native-and-kubernetes-observability-panel-the-state-of-union-bartek-plotka-red-hat-liz-fong-jones-honeycomb-josh-suereth-google-frederic-branczyk-polar-signals-rags-srinivas-infoq) - Bartek Plotka, Red Hat; Liz Fong-Jones, Honeycomb; Josh Suereth, Google; Frederic Branczyk, Polar Signals; Rags Srinivas, InfoQ
```

Comment:
How the people are listed for this talk is a slightly different format than the other talks.

Reply:
good catch.